tests/integration/questionnaire/test_page_titles.py

```diff
@@ -0,0 +1,13 @@
+from tests.integration.integration_test_case import IntegrationTestCase
+
+
+class TestPageTitles(IntegrationTestCase):
+    def test_should_have_error_in_page_title_when_fail_validation(self):
+        # Given
+        self.launchSurvey("test_checkbox")
+        # When
+        self.post()
+        # Then
+        self.assertEqualPageTitle(
+            "Error: Which pizza toppings would you like? - Other input fields"
+        )
```

Comment:
Sorry, not sure how I missed this, but there is already a test file that tests page titles - `test_questionnaire_page_titles.py`, this test should be in that file.